monasca-alarms/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM alpine:3.5
+ARG MON_CLIENT_VERSION="1.6.0"
```

Comment:
There is newer version on master.

Reply:
Done

Reply:
Sorry, but for now we have to stick with `1.6.0`. `1.7.0` comes with breaking changes in the API hence this PR would require too much work to complete. `1.7.0` should be considered as future enhancement.